src/components/List/ListItem/Variants/SelectAllSuggestions/SelectAllSuggestions.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { bem } from '../../../../../utils';
+import { Props as ListItemProps } from '../../ListItem';
+import styles from './SelectAllSuggestions.scss';
+import { Checkbox } from '../../../../Checkbox';
+import { SPACE_KEY } from '../../../../../constants';
+
+export interface Props
+    extends Omit<ListItemProps, 'onClick' | 'onChange' | 'value' | 'highlightContext'> {
+    /** A function to be called if the item is clicked */
+    onChange: (e: React.SyntheticEvent<HTMLInputElement | HTMLLIElement>, value) => void;
```

Comment:
It's not absolutely necessary, but maybe it makes sense to make this `onChange` optional? if we plan to use this component in our current `Dropdown`, `onChange` will become obsolete because to prevent the dropdown from closing during multiselection when using focusable elements, preventDefault was added to the list's `onClick`, which prevents this `onChange` from firing. Also, the `onChange` from the dropdown handles all selections.
What do you think? You can test it in the `Dropdwon` after rebasing your branch to the latest version.